shared/messaging/proto/addressbook_at_height.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto3";
+
+package pocket;
+
+option go_package = "github.com/pokt-network/pocket/shared/messaging";
+
+message AddressBookAtHeight {
+  repeated NetworkPeer network_peers = 1;
+  uint64 height = 2;
+}
+
+message NetworkPeer {
+  bytes public_key = 1;
+  bytes address = 2;
```

Comment:
Can we change this into a string.

The fact that we're not consistent is legacy...

Reply:
This code has been deprecated already :) fail fast they say. It doesn't get faster than this IMHO